### PR TITLE
include freetype-config in freetype-dev

### PR DESCRIPTION
**Pull request by stephengroat**
required for texlive

#1169

### CODE REVIEW

File: packages/freetype/build.sh [hunk 1]
```diff
@@ -4,6 +4,6 @@ TERMUX_PKG_VERSION=2.8
 TERMUX_PKG_SRCURL=https://downloads.sourceforge.net/project/freetype/freetype2/${TERMUX_PKG_VERSION}/freetype-${TERMUX_PKG_VERSION}.tar.bz2
 TERMUX_PKG_SHA256=a3c603ed84c3c2495f9c9331fe6bba3bb0ee65e06ec331e0a0fb52158291b40b
 TERMUX_PKG_DEPENDS="libbz2, libpng"
-TERMUX_PKG_RM_AFTER_INSTALL="bin/freetype-config share/man/man1/freetype-config.1"
+TERMUX_PKG_INCLUDE_IN_DEVPACKAGE=="bin/freetype-config share/man/man1/freetype-config.1"
```
Comment (Grimler91) [inline packages/freetype/build.sh]: == should be =

**Comment by Grimler91** [comment]
What was the error when building without it? Do you have a log?

**Comment by stephengroat** [comment]
@Grimler91 no error currently.

in #1169, i've changed `build-package.sh` to use deb instead of compiling all dependencies. significantly speeds up builds, but some of the dependencies aren't explicitly listed right now

**Comment by Grimler91** [comment]
@stephengroat ah, right! I see.

**Comment by fornwall** [comment]
Thanks! Merged and the `==` typo fixed. The revision-bumped freetype package is now available in the apt repository.

File: packages/freetype/build.sh (hunk 1 above)
Comment (stephengroat) [inline packages/freetype/build.sh]: thanks for that, completely missed it

**Comment by stephengroat** [comment]
@fornwall thanks for fixing the typo, completely missed it

**Pull request merged**